includes/adapter/OrderLineFromWooCommerce.php

```diff
@@ -52,15 +52,40 @@ protected function prepareProductData(WC_Order_Item $wcOrderItem): array
         $wcItemData = $wcOrderItem->get_data();
         $wcProduct  = $wcOrderItem->get_product();
```

Comment:
als je hier (en evt op andere plekken) `$wcOrderItem->get_product() ?: null` doet kan je in `getDataFromProduct` wel gewoon de type `?WC_Product` gebruiken

Reply:
wc product kan helaas ook null teruggeven, en dan faalt ?: volgens mij, ik zal er naar kijken

Reply:
Nee kan gewoon, het werkt op "falsy", niet alleen op `false`